### PR TITLE
Support 'sn' alias of 'last' for surname

### PR DESCRIPTION
**Pull request by cdchapman**
What would you think about supporting an LDAP-style alias for 'last'?

### CODE REVIEW

**Comment by t-woerner** [comment]
Yes, it should be supported. Please also add the alias to the README of the ipauser module.

File: tests/user/test_users.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@
       users:
       - name: user1
         givenname: user1
-        last: Last
+        sn: Last
```
Comment (t-woerner) [inline tests/user/test_users.yml]: Only change one for the test please. This way the test will make sure that `last` and `sn` are the same.

**Comment by cdchapman** [comment]
I made those updates. Let me know if I need to change anything else.

**Comment by t-woerner** [comment]
That looks good. Thanks.

**Pull request merged**